cves/CVE-2019-14232.yml

```diff
@@ -65,25 +69,29 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-- commit: c3289717c6f21a8cf23daff1c78c0c014b94041f
-  note:
-- commit: c23723a1551340cc7d3126f04fcfd178fa224193
-  note:
-- commit: 42a66e969023c00536256469f0e8b8a099ef109d
-  note:
+  - commit: c3289717c6f21a8cf23daff1c78c0c014b94041f
+    note:
+  - commit: c23723a1551340cc7d3126f04fcfd178fa224193
+    note:
+  - commit: 42a66e969023c00536256469f0e8b8a099ef109d
+    note:
+  - commit: 7f65974f8219729c047fbbf8cd5cc9d80faefe77
+    note:
 vccs:
-- commit: 201017df308266c7d5ed20181e6d0ffa5832e3e9
-  note: This VCC was discovered automatically via archeogit.
-- commit: 3b774583711e39dae7a5cfde314288f8019f59c6
-  note: This VCC was discovered automatically via archeogit.
-- commit: 92053acbb9160862c3e743a99ed8ccff8d4f8fd6
-  note: This VCC was discovered automatically via archeogit.
-- commit: 321e94fa41b121f65c02119c02098df327bbd569
-  note: This VCC was discovered automatically via archeogit.
-- commit: d693074d431c50e4801dd6bf52525ce1436358f0
-  note: This VCC was discovered automatically via archeogit.
-- commit: ac4faa6dc33407c93566884e53fa5d8ef44c0a2a
-  note: This VCC was discovered automatically via archeogit.
+  - commit: 201017df308266c7d5ed20181e6d0ffa5832e3e9
+    note: This VCC was discovered automatically via archeogit.
+  - commit: 3b774583711e39dae7a5cfde314288f8019f59c6
+    note: This VCC was discovered automatically via archeogit.
+  - commit: 92053acbb9160862c3e743a99ed8ccff8d4f8fd6
+    note: This VCC was discovered automatically via archeogit.
+  - commit: 321e94fa41b121f65c02119c02098df327bbd569
+    note: This VCC was discovered automatically via archeogit.
+  - commit: d693074d431c50e4801dd6bf52525ce1436358f0
+    note: This VCC was discovered automatically via archeogit.
+  - commit: ac4faa6dc33407c93566884e53fa5d8ef44c0a2a
+    note: This VCC was discovered automatically via archeogit.
+  - commit: c22212220a7900173358a1f16179dcfc9e03de78
```

Comment:
I added short comments for how vccs impacted corresponding vulnerability. Not sure if its required but would be nice for reference.